lms/templates/login.html

```diff
@@ -1,12 +1,13 @@
 <%inherit file="main.html" />
```

Comment:
This file isn't going to be rendered because your site uses the combined Logistration page (the Backbone.js page) instead of this old HTML template. That being said, these changes aren't causing any problems, so you can leave it if you'd like. Or you can just nuke this file all together.

Reply:
I like the idea of removing this template.  Will do.